swap/protocol_test.go

```diff
@@ -36,8 +36,14 @@ import (
 	contract "github.com/ethersphere/swarm/contracts/swap"
 	"github.com/ethersphere/swarm/p2p/protocols"
 	p2ptest "github.com/ethersphere/swarm/p2p/testing"
+	colorable "github.com/mattn/go-colorable"
 )
 
+func init() {
+	log.PrintOrigins(true)
+	log.Root().SetHandler(log.LvlFilterHandler(log.Lvl(*loglevel), log.StreamHandler(colorable.NewColorableStderr(), log.TerminalFormat(true))))
```

Comment:
Why did you choose to make it *loglevel here, instead of writing `2` or `3`?

Reply:
So I can control the test verbosity from only one place.

Reply:
This is how we usually do this for testing. What this does is that we can add `go test .... -loglevel X` to tests and manually set it. So this is good like this.